### ISSUE TITLE
DetailsList: High Contrast Support

### ISSUE THREAD

**Issue by Boris-Em** [OfficeDev/office-ui-fabric-react#2856]
### Bug Report

- __Package version(s)__: 5.0
- __Browser and OS versions__: Edge

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Yes

Requested priority: Low

#### Describe the issue:

The selection arrow is not accessible in high contrast mode.

#### Actual behavior:

![detailslist](https://user-images.githubusercontent.com/4613643/30461613-6eec4fa4-9974-11e7-9897-a49ded3dc53c.png)

#### Expected behavior:

The selection arrow on the left of a line should be more visible in high contrast mode.

**Comment by kristoferbrown** [comment]
Fixed by #2589
